R/validations.R

```diff
@@ -247,13 +252,14 @@ validate_no_intersection <- function(x, y, msg) {
 
 #' Validates that dataset contains specific variable
 #'
-#' @description `r lifecycle::badge("stable")`
-#' @param data a `data.frame`
+#' `r lifecycle::badge("stable")`
```

Comment:
Won't this push the second paragraph to `@details` section?

```suggestion
#' @description `r lifecycle::badge("stable")`
```